metricbeat/module/kubelet/_meta/fields.yml

```diff
@@ -0,0 +1,9 @@
+- key: kubelet
+  title: "kubelet"
+  description: >
+    kubelet Module
```

Comment:
Can you add `[]beta` flag and `short_config: false` here? See https://github.com/elastic/beats/blob/master/metricbeat/module/docker/_meta/fields.yml#L7 for an example.